cucumber-expressions/java/src/main/java/io/cucumber/cucumberexpressions/Ast.java

```diff
@@ -1,14 +1,17 @@
 package io.cucumber.cucumberexpressions;
 
+import static java.util.Objects.requireNonNull;
+import static java.util.stream.Collectors.joining;
+
+import java.util.Collections;
 import java.util.List;
 import java.util.Objects;
 import java.util.StringJoiner;
 
-import static java.util.Arrays.asList;
-import static java.util.Objects.requireNonNull;
-import static java.util.stream.Collectors.joining;
+import org.apiguardian.api.API;
 
-final class Ast {
+@API(status = API.Status.INTERNAL)
```

Comment:
This should be experimental. The eclipse plugin is not internal to the cucumber expressions library.